packages/TelegramClientTests-Core.package/TCTCChatsTests.class/instance/testSortingUnknownPositions.st

```diff
@@ -4,7 +4,7 @@ testSortingUnknownPositions
 	| chat |
 
 	chat := self chats last.
-	chat position: nil.
+	chat position: -1.
```

Comment:
I am not sure about this, maybe @rsommerfeld knows it better but I think only positions of nil are considered unknown. Actually, Tdlib might return negative positions for some chats, if I remember correctly.

Reply:
I'm not sure if positions can ever be negative. My intuition would be they can't (chat ids definitely can be negative though).

Using nil or -1 as "position is not known" is equally bad as magic value. I introduced a class-side constant expressing the meaning of -1 as "unknown position" and introduced a test that tests, that there is never a chat on the Telegram Test Data Center with negative positions. In case that ever fails, at least it encodes our assumption - for now it runs through 😄

Reply:
Okay nice, I've mistaken the chat ids with chat positions then, probably